### DISCUSSION TITLE
Ink

### DISCUSSION THREAD

**Discussion by markdav-is** [markdav-is/Hexstor#7] [General]
Hexstor is a web-game so it's C#, Blazor.  I'm pulling in the Ink libraries from Inkle studios  (https://www.inklestudios.com/) because it's C# we can use libraries similar to Unity or Gadot (but the generic versions). The basic flow is story/encounter repeat.  So choices in the story can determine the elements of the encounter, and actions in the encounter can affect the story.  It would be great to have graphics to go along with the story.  Artists could dream up various factions or races and writers could weave those together.

**Comment by Vexmage** [comment]
I've done some paid game development writing work for Paizo's Starfinder and Pathfinder 2 series, as well as run my own pathfinder games with custom content (writing, rules, and artwork) so I might be able to contribute there as well.  Do we have an overall plot and setting for this game?
